artifactory/commands/python/python.go

```diff
@@ -71,7 +72,24 @@ func (pc *PythonCommand) Run() (err error) {
 		err = errorutils.CheckError(pythonModule.RunInstallAndCollectDependencies(pc.args))
 	} else {
 		// Python native command
-		err = gofrogcmd.RunCmd(pc)
+		for k, v := range pc.GetEnv() {
+			if err := os.Setenv(k, v); err != nil {
+				return err
+			}
+		}
+
+		cmd := pc.GetCmd()
+		errBuffer := bytes.NewBuffer([]byte{})
+		multiWriter := io.MultiWriter(os.Stderr, errBuffer)
+		cmd.Stderr = multiWriter
+		cmd.Stdout = os.Stdout
+
+		err = cmd.Run()
+		if err != nil {
+			if buildInfoUtils.IsForbiddenOutput("pip", errBuffer.String()) {
+				err = errors.Join(err, buildInfoUtils.NewForbiddenError())
+			}
+		}
```

Comment:
The above code should be mioved into in the build-info-go module. If it isn't place there, this functionality will not work for Pythin when collecting build-info.
It is true that when not collecting build-info for python, the code flow ends here and doesn't continue into build-info-go, but having this code there, will allow you to invoke it from here as well.

Reply:
Eyal, In case we collect build info, the whole installation have different logic which apparently adding the stderr to the err, so in this case it will work in the flow of pip install + collecting dependencies for build-info, I security wrapper we recognize either a 403 in the error string OR forbidden type error.

I don't want to use the same logic to run installation command for the build info, as it can break existing behavior, which I avoided in this PRs.